qubesmanager/settings.py

```diff
@@ -481,7 +481,10 @@ def __init_basic_tab__(self):
             QtGui.QRegularExpressionValidator(
                 QtCore.QRegularExpression(
                     "[a-zA-Z0-9_-]*",
-                    QtCore.QRegularExpression.PatternOption.CaseInsensitiveOption,
+                    # fmt: off
```

Comment:
why turning off formatting in such an awkward place (the middle of function declaration)? I do not like black myself, but I don't think this is a good approach to deal with its... opinions